docs/WifiPerformance.md

```
@@ -60,980 +60,912 @@
+    TX: 54438 bytes (200 packets)
+    signal: -27 dBm
+    rx bitrate: 1200.9 MBit/s 80MHz HE-MCS 11 HE-NSS 2 HE-GI 0 HE-DCM 0
+    tx bitrate: 1200.9 MBit/s 80MHz HE-MCS 11 HE-NSS 2 HE-GI 0 HE-DCM 0
+    
+```
 
-### Realtek 8811AU
+#### Comfast CF953AX
 
-<img src=https://stuff.armbian.com/netbox/media/devicetype-images/RTL8812AU.png>
+<img src=https://stuff.armbian.com/netbox/media/devicetype-images/MT7921AU.png>
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)